coaster/sqlalchemy.py

```diff
@@ -28,6 +28,9 @@ class Query(BaseQuery):
 
     def one_or_none(self):
         """
+        #: ..deprecated
+        #: Use SQL-Alchemy's native one_or_none (available since 1.0.9)
+
```

Comment:
No, you have to *remove* this function entirely because it's now in the base class we're deriving from. Instead change the SQLAlchemy dependency in `setup.py` to `>=1.0.9`.